### DISCUSSION TITLE
Is anybody using the standalong mode (OpenapiFirst.app)

### DISCUSSION THREAD

**Discussion by ahx** [ahx/openapi_first#168] [General]
The [standalone mode](https://github.com/ahx/openapi_first#standalone-usage) was mainly inspired by [connexion](https://github.com/spec-first/connexion). I like the approach, but I am wondering if others are using it. If not, I would like to remove that part in order to make the project easier to maintain.

**Comment by twe4ked** [comment]
We're not. We're likely to look at using the middlewares as part of a Rails app at some point, but likely not standalone. I feel like if you've got to write any code in a Rackup file it's probably not too much overhead to make you're own app (I'm +1 for removing it). Might be nice to have a minimal example of what to do instead of using `OpenapiFirst.app`.

**Comment by ahx** [comment]
Hi. I have removed `Openapi.app` and other stuff and have just released [1.0.0.beta1](https://github.com/ahx/openapi_first/blob/main/CHANGELOG.md#100beta1). The main change besides removing the standalone mode is that query and path parameters now support `explode: true` and all `style` options as defined in the OpenAPI specification. 
I would be happy if you would try it out.
